ra2ce/network/network_simplification/nx_to_snkit_network_converter.py

```diff
@@ -49,6 +51,7 @@ def convert(self) -> SnkitNetwork:
         """
         # Extract graph values
         _crs = self.networkx_graph.graph.get("crs", None)
+        self.networkx_graph = add_x_y_to_nodes(self.networkx_graph)
```

Comment:
What is the reason for and effect of this change?

Reply:
I added a general comment. Can not recall its relation to the issue. Tried to check it by commenting this line. Tests passed.